config/plugins

```diff
@@ -7,6 +7,7 @@
 #
 # To see the help docs for a particular plugin, run 'haraka -h plugin.name'
 
+status
```

Comment:
```suggestion
# status
```

Open question: should this plugin be enabled by default? We purposefully keep the enabled-by-default plugins to a minimum. I lean towards not enabled by default.

I like and expect to use this plugin, and there's a good chance that someone extends it with web hooks. But that's not a good enough reason to have it enabled by default. Shucks, we don't even have TLS enabled by default.

Reply:
Definitely no need to enable this by default, this is reason why this is done as plugin (vs #2428). Apache's mod_status or nginx status page is not enabled by default also

I am going to implement web hooks too eventually. Also extending plugin (and haraka) with hooks like command_help and command_exec to just move "--qlist, --qstat, --qunstick..." commands inside plugin and remove logic from core would be nice.